Cargo.toml

```diff
@@ -33,5 +33,5 @@ either = "1.0.3"
 error-chain = "0.11.0"
 inflections = "1.1.0"
 quote = "0.3.15"
-svd-parser = "0.6"
+svd-parser = { git = "https://github.com/japaric/svd" }
```

Comment:
Git version here means we can't publish this to crates.io, perhaps we need to an svd-parser release?

Reply:
(cc. @Emilgardis)

Reply:
Yeah, this is gross.  The problem is that svd-parser master hasn't been published in a while; the `derive_from` method that we need was added a long time ago but still isn't available.

Reply:
Mhmm, I am [almost completely](https://github.com/japaric/svd/issues/46) at fault for that sorry :-/
Have been looking at it again today, but I think I need to learn a bunch more about SVD files to get it finished.

Reply:
No worries; I know exactly how it goes :-)  I have my own branch that I've been using in the meantime, so it's not a critical blocker for my projects.